src/Service/BoolParser.php

```diff
@@ -16,7 +16,12 @@ class BoolParser implements BoolParserInterface
      */
     public function parse($value): bool
     {
-        $normalizedValue = \trim(\strtolower($value));
-        return \in_array($normalizedValue, ['yes', 'on', 'true', '1']);
+        if (\is_bool($value)) {
+            return $value;
+        }
+        if (\is_string($value)) {
```

Comment:
Did you change the codestyle rules?
I personally prefer more newlines:
```
        if (\is_bool($value)) {
            return $value;
        }

        if (\is_string($value)) {
            $value = \trim(\strtolower($value));
        }

        return \in_array($value, ['yes', 'on', 'true', '1', 1]);
```

Reply:
No did not change anything on the codestyle rules.

Reply:
I thought we forced that a long time ago. But I guess this made problems with block ending at the end of blocks or so....

Reply:
Yes that is possible